### PR TITLE
Comments: Make header in single post view sticky

### LINKED ISSUE #20844: Comments: make header in single post view sticky
While interviewing users, some had trouble distinguishing the single post view from the default view. Sticking the header component to the top of the viewport will help orient users.

screen shot 2017-12-14 at 3 28 30 pm

Example of sticky header:

screen shot 2017-12-14 at 3 29 03 pm

### PR DESCRIPTION
**Pull request by Copons**
Fix #20844

Make header in single post view of Comments Management sticky.

dec-18-2017 18-05-33

### CODE REVIEW

File: client/my-sites/comments/style.scss [hunk 1]
```diff
@@ -75,6 +75,15 @@
 	}
 }
 
+.comment-list__header {
+	overflow: visible;
+
+	&.is-sticky .header-cake {
+		box-shadow: 0 2px 5px rgba(0, 0, 0, 0.1), 0 0 56px rgba(0, 0, 0, 0.075);
+		margin: 9px 0;
```
Comment (rodrigoi) [inline client/my-sites/comments/style.scss]: this margin leaves a gap between the sticky header and the Calypso header.

Comment (gwwar) [reply client/my-sites/comments/style.scss]: +1 for removing the gap ^^

**Comment by megsfulton** [comment]
Another +1 for removing the gap. Should look/behave the same way the sticky headers do on Stats, Media Library, and Themes.

**Comment by Copons** [comment]
@rodrigoi Awesome catch!
I was testing on a screen size where the sticky header has indeed a margin, and misread the CSS.

With the last change, this headers behave the same way as the others: the top margin is only there between 660 and 960; smaller screens don't have stickiness, and larger don't have the margin.

**Pull request merged**